source/usage-examples/bulkWrite.txt

```diff
@@ -0,0 +1,93 @@
+=======================
+Perform Bulk Operations
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+:node-api:`collection.bulkWrite <Collection.html#bulkWrite>` lets you
+perform bulk write operations against a *single* collection. With
+``collection.bulkWrite()``, you specify a list of operations to perform and
+MongoDB then executes those operations in bulk. Bulk write supports
+``insertOne``, ``updateOne``, ``updateMany``, ``deleteOne``,
+``deleteMany``, and ``replaceOne`` operations. Refer to the method
+documentation for full details.
+
+``bulkWrite()`` accepts the following parameters:
+
+- ``operations``: specifies the bulk operations to
+  perform. Each operation is passed to ``bulkWrite()`` as an object in
+  an array.
+
+- ``options``: *optional* settings that affect the execution
+  of the operation, such as :manual:`write concern
+  </reference/write-concern>` and order.
+
+  By default, MongoDB executes bulk operations in the specified order
+  (i.e. serially). When executing operations in order, if an error occurs
+  during the processing MongoDB returns without processing any of the
```

Comment:
comma after the first processing here? my brain got all loopy when i read this the first time and had to figure out where to pause.

Reply:
that was not a good sentence. ugh. rephrased